tests/unit/conftest.py

```diff
@@ -19,6 +20,7 @@ def config() -> Dict:
         "provider": "generic",
         "issuer_url": "http://example.com",
         "secret_backend": "relation",
+        "scope": "profile email  address  phone",
```

Comment:
are multiple spaces intentional?
```suggestion
        "scope": "profile email address phone",
```